flexbox/src/main/java/com/google/android/flexbox/FlexboxLayout.java

```diff
@@ -114,6 +115,11 @@
      */
     private int mAlignContent;
 
+    /**
+     * The current value of the {@link }
```

Comment:
Isn't there an issue in the comment here?

Reply:
Oh good catch. I'll fix in a follow up PR